DependencyInjection/Configuration.php

```diff
@@ -39,7 +39,7 @@ public function getConfigTreeBuilder()
                     ->scalarNode('cache')->defaultTrue()->end()
                     ->scalarNode('cache_timeout')->defaultValue(60000)->end()
                     ->scalarNode('width')->defaultNull()->end()
-                    ->scalarNode('object_manager')->defaultValue(1)->end()
+                    ->scalarNode('object_manager')->defaultNull()->end()
```

Comment:
Why was 1 the default value before? Coming to think of it, why did the object_manager node even exist in the first place?

Reply:
Yeah, a default of 1 makes no sense at all.

I think the changes you made are good with the ability to have either a "global" custom manager or per instance.